zebra/zebra_fpm_netlink.c

```diff
@@ -100,6 +101,49 @@ static size_t af_addr_size(uint8_t af)
 	}
 }
 
+/*
+ * We plan to use RTA_ENCAP_TYPE attribute for VxLAN encap as well.
+ * Currently, values 0 to 8 for this attribute are used by lwtunnel_encap_types
+ * So, we cannot use these values for VxLAN encap.
+ */
+enum fpm_nh_encap_type_t {
+	FPM_NH_ENCAP_NONE = 0,
+	FPM_NH_ENCAP_VXLAN = 100,
```

Comment:
How did you come up with 100?  Are we in danger of collision here?

Reply:
Currently, values 0 to 8 are used as I have stated in the comment. Any suggestion value we can use for VxLAN encap if not 100?